packages/backend-common/src/database/postgres.ts

```diff
@@ -95,3 +95,35 @@ function requirePgConnectionString() {
     throw new Error(`${message}\n${e.message}`);
   }
 }
+
+/**
+ * Creates the missing Postgres database if it does not exist
+ *
+ * @param dbConfig The database config
+ * @param databases The name of the databases to create
+ */
+export async function ensurePgDatabaseExists(
+  dbConfig: Config,
+  ...databases: Array<string>
+) {
+  const admin = createPgDatabaseClient(dbConfig, {
+    connection: {
+      database: 'postgres',
+    },
+  });
+  const ensureDatabase = async (database: string) => {
+    const result = await admin
+      .from('pg_database')
+      .where('datname', database)
+      .count<Record<string, { count: string }>>();
+
+    if (parseInt(result[0].count, 10) > 0) {
+      return;
+    }
+
+    await admin.raw(`CREATE DATABASE ??`, [database]);
+  };
+
+  await Promise.all(databases.map(ensureDatabase));
+  await admin.destroy();
```

Comment:
should this be in a try/finally to ensure it happens even if the above line fails? not a huge deal since it's just once at startup, but still

Reply:
Yeah, forgot about it. Fixed.

Reply:
@lowjoel would you happen to know what level of database access is required after the PostgreSQL database has been created?